docs/en/docs/sql-manual/sql-functions/string-functions/concat_ws.md

```diff
@@ -28,10 +28,10 @@ under the License.
 ### Description
 #### Syntax
 
-`VARCHAR concat ws (VARCHAR sep, VARCHAR str,...)`
+`VARCHAR concat ws (VARCHAR sep, VARCHAR str,...) or VARCHAR concat_ws(VARCHAR sep, ARRAY array)`
 
 
-Using the first parameter SEP as a connector, the second parameter and all subsequent parameters are spliced into a string.
+Using the first parameter SEP as a connector, the second parameter and all subsequent parameters(or all string in an ARRAY) are spliced into a string.
 If the separator is NULL, return NULL.
 ` The concat_ws` function does not skip empty strings, but NULL values.
```

Comment:
I think this line change to "The `concat_ws` function does not skip empty strings, it skips NULL values" is more more acurrate.

Reply:
OK